Automattic/sensei
# Change "Install sample course" redirect to the imported course

### Changes proposed in this Pull Request

@donnapep asked [here](https://github.com/Automattic/sensei/pull/3507#issuecomment-674908604) to redirect the user to the imported course after the sample course installing. So I added the completed IDs (for importer and exporter) to our API.

If we don't want to add this extra complexity, @donnapep said that [we could skip this](https://github.com/Automattic/sensei/pull/3507#issuecomment-675603913).

### Testing instructions

* Go to the Setup Wizard, and navigate until the ready step.
* Click on `Install a sample course`.
* Wait until the installation and make sure you're redirected to the created course page.
